index.html

```diff
@@ -134,19 +134,40 @@ <h3>Peanut butter</h3>
     </div>
 
     <div class="card card-hz">
-            <div class="card-img-container card-img-hz"><img class="card-img" src="./components/images/peanut-product.jpg" alt="card">
-            </div>
-            <div class="card-desc">
-                <h3>Peanut butter</h3>
-                <p>by American Garden</p><span class="card-price">Rs 499</span><span class="card-price-line-through">Rs
-                    999</span>
-                <div class="card-btns">
-                    <button class="card-cart btn btn-primary">add to cart</button><button
-                        class="card-wishlist btn btn-secondary">wishlist</button>
-                </div>
+        <div class="card-img-container card-img-hz"><img class="card-img" src="./components/images/peanut-product.jpg"
+                alt="card">
+        </div>
+        <div class="card-desc">
+            <h3>Peanut butter</h3>
+            <p>by American Garden</p><span class="card-price">Rs 499</span><span class="card-price-line-through">Rs
+                999</span>
+            <div class="card-btns">
+                <button class="card-cart btn btn-primary">add to cart</button><button
+                    class="card-wishlist btn btn-secondary">wishlist</button>
             </div>
         </div>
+    </div>
 
+    <!-- image component -->
+    <h3>responsive image</h3>
```

Comment:
go for review with simple or related changes, so revw can understand and find things which need your  attention